protocol/x/vault/keeper/orders.go

```diff
@@ -417,19 +414,6 @@ func (k Keeper) PlaceVaultClobOrder(
 	order *clobtypes.Order,
 ) error {
 	err := k.internalPlaceVaultClobOrder(ctx, vaultId, order)
```

Comment:
nit: without a need to emit indexer events, can we remove `internalPlaceVaultClobOrder` and just have that logic in `PlaceVaultClobOrder`

Reply:
Would rather leave that out of a revert.

Reply:
Done!